packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -674,7 +667,9 @@ export default function ScheduleDetails({ modalProps, actions, id }) {
                 type="bare"
                 style={{
                   color:
-                    state.transactionsMode === 'linked' ? colors.b4 : colors.n7,
+                    state.transactionsMode === 'linked'
+                      ? theme.tableTextEditingBackground
+                      : 'inherit',
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏wrong color

before
<img width="362" alt="Screenshot 2023-08-12 at 09 31 38" src="https://github.com/actualbudget/actual/assets/886567/69a0d69b-45d4-4b47-b101-0e73f0c777bc">


after
<img width="362" alt="Screenshot 2023-08-12 at 09 31 35" src="https://github.com/actualbudget/actual/assets/886567/ae81e637-68e4-4224-9735-40cc38ace95b">